### PR TITLE
Module 6

### PR DESCRIPTION
**Pull request by vladSHOKO**
Rewritten code with cicle 'foreach'

### CODE REVIEW

**Comment by vladSHOKO** [comment]
@Lehersun module_6 ready to check

File: module_6/library.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+$title = 'Библиотека программиста';
+$countOfAuthors = count($library['authors']);
+$red = (bool) rand(0, 1);
+
+if ($red) {
+    $red = "red";
+}
+else {
+    $red = " ";
+}
```
Comment (Lehersun) [inline module_6/library.php]: check code style here. `else` should be on the same line with `}`
like `} else {`

**Review by Lehersun — CHANGES_REQUESTED** [review]

**Review by vladSHOKO — COMMENTED** [review]
Code style remade with help of hotkeys CTRL+ALT+L

**Comment by vladSHOKO** [comment]
@Lehersun

**Comment by Lehersun** [comment]
@vladSHOKO, I don't know what's really going wrong, but I still see coded style problems. Let's make it together tommorow

**Comment by vladSHOKO** [comment]
@Lehersun task ready to check

**Comment by Lehersun** [comment]
Well done

**Pull request merged**